frontend/src/pages/Artwork/Artwork.tsx

```diff
@@ -131,3 +150,47 @@ export default function Artwork() {
     </Switch>
   );
 }
+
+interface GroupDrawerProps {
```

Comment:
I haven't used any interfaced in my components, this is smart.

Reply:
Yeah! It gives you type completion and is a bit looser than stating `type GroupDrawerProps`. Definitely helps communicate intention as well.